apps/files_versions/lib/Versions/IVersionBackend.php

```diff
@@ -78,4 +79,12 @@ public function read(IVersion $version);
 	 * @since 15.0.0
 	 */
 	public function getVersionFile(IUser $user, FileInfo $sourceFile, $revision): File;
+
+	/**
+	 * Get the revision for a node
+	 *
+	 * @param \OC\Files\Node\Node $node
+	 * @return int
```

Comment:
```suggestion
	 * Get the revision for a node
	 *
	 * @since 32.0.0
```

The types are not needed as you already have the strong types, but you need to add the version in which this will be added. This also needs to be added to https://docs.nextcloud.com/server/latest/developer_manual/app_publishing_maintenance/app_upgrade_guide/upgrade_to_32.html.

Reply:
I have applied the suggested code change but the changelog you mention I don't know in which file to do it and if I have permission to do it.

Reply:
That's in https://github.com/nextcloud/documentation/blob/master/developer_manual/app_publishing_maintenance/app_upgrade_guide/upgrade_to_32.rst, you can also fork the repo and open a PR over there.

Reply:
Ready @provokateurin  https://github.com/nextcloud/documentation/pull/13386